base_site/index.html

```diff
@@ -1,96 +1,222 @@
     </div>
-  </div>
-</header>
-<div class="bd-container">
-  <main id="main-content" class="bd-main">
-    <div class="bd-content">
-      <div class="bd-article-container">
-        <article class="bd-article">
-            <div class="card">
-              <div class="card-body">
-                <h4 class="card-title">InQuanto</h4>
-                <p>A Python library for chemistry and material science workflows on quantum computers.</p>
-                <a href="inquanto/index.html">See InQuanto docs</a>
+    <main class="main-wrapper">
+      <header class="section_hero">
+        <div class="padding-global">
+          <div class="container">
+            <div class="padding-section-small">
+              <div class="component_header">
+                <div id="w-node-_27207c92-4407-f8bd-ceae-3c0aeacb91e5-c2a09c1d" class="header_wrap">
+                  <div class="header_logo-wrap"><img src="images/logo.svg" loading="lazy" alt="" class="image-fit-contain"></div>
+                  <h1 class="heading-style-h1 text-color-gray">Technical Documentation</h1>
+                </div>
+                <div id="w-node-_190ab80b-6ed6-7351-35be-2accb7933157-c2a09c1d" class="header_wrap is-home"><img src="images/q.svg" loading="lazy" alt="" class="image-fit-contain"></div>
               </div>
             </div>
-            <div class="card">
-              <div class="card-body">
-                <h4 class="card-title">H-Series</h4>
-                <p>Ion-trap hardware, H-Series, powered by Honeywell.</p>
-                <a href="h-series/index.html">See H-Series docs</a>
+          </div>
+        </div>
+      </header>
+      <section class="section_products-list">
+        <div class="padding-global">
+          <div class="container">
+            <div class="padding-section-small">
+              <div class="product_list-wrap">
+                <div class="w-layout-grid product-list_component is-first">
+                  <div class="product-list_content">
+                    <div class="product_logo-wrap"><img src="images/nexus-wordmark.svg" loading="lazy" alt="" class="image-fit-contain"></div>
+                    <div class="spacer-small"></div>
+                    <p class="product_description">Cloud platform access and manage our full-stack</p>
+                    <div class="spacer-medium"></div>
+                    <a href="https://docs.quantinuum.com/nexus/index.html" class="product_link-wrap w-inline-block">
+                      <p class="product_link-text">Get Started with Nexus</p>
+                      <p class="product_description is-list">Our platform for accessing and managing the highest performance full-stack</p>
+                    </a>
+                    <div class="spacer-small"></div>
+                    <a href="https://docs.quantinuum.com/nexus/user_guide/sign_up.html" class="product_link-wrap w-inline-block">
+                      <p class="product_link-text">Tutorials and Documentation</p>
```

Comment:
```suggestion
                    <a href="https://docs.quantinuum.com/nexus/user_guide/sign_up.html" class="product_link-wrap w-inline-block">
                      <p class="product_link-text">Get Started with Nexus</p>
                      <p class="product_description is-list">Our platform for accessing and managing the highest performance full-stack</p>
                    </a>
                    <div class="spacer-small"></div>
                    <a href="https://docs.quantinuum.com/nexus/index.html" class="product_link-wrap w-inline-block">
                      <p class="product_link-text">Tutorials and Documentation</p>
```

Swap around the "Get started" and "Tutorials and Documentation" links - the signup page should be the "Get started" one, the main docs subsite should be "Tutorials and Documentation".